lib/modem_key_mgmt/modem_key_mgmt.c

```diff
@@ -90,7 +90,7 @@ static int key_fetch(nrf_sec_tag_t tag,
 	cmee_enable(&cmee_was_active);
 
 	err = nrf_modem_at_cmd(scratch_buf, sizeof(scratch_buf),
-			       "AT%%CMNG=2,%d,%d", tag, cred_type);
+			       "AT%%CMNG=2,%u,%u", tag, cred_type);
```

Comment:
`nrf_sec_tag_t` is unsigned, so using `%u` is understandable, but shouldn't the format specifier for `modem_key_mgmt_cred_type` be left as is? It's `enum` so it is signed int.

Reply:
I agree with that

Reply:
Fixed